samples/drivers/kscan_ft5336/src/main.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020 Mark Olsson <mark@markolsson.se>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
```

Comment:
Please make it generic, the sample should be for the driver API not for specific driver.

Reply:
Isn't the other sample specific to another driver? I guess it's better to change it to be a touch sample?

Reply:
> Isn't the other sample specific to another driver?

Indeed. Could you rename your sample to something like kscan_touch and make it more generic?